### ISSUE TITLE
No support for IPV6 tunnel endpoints

### ISSUE THREAD

**Issue by ernstjo** [bbonev/eoip#10]
There is no IPv6 support for tunnel end points. It occurs an error "invalid ip address" if you use IPv6 addresses for **local** and **remote** ip address. Thus, not compatible with Mikrotik IPv6 EOIP. Please extend support.

**Comment by bbonev** [comment]
True, IPv6 is not implemented at all. I am currently working on DKMS maybe IPv6 comes afterwards?

In case you can contribute protocol specification or code, you are welcome.

**Comment by zoigecyb** [comment]
Maybe it's not a very good solution, but i tried ROS v7 on older hardware and it causes drastically high cpu load...
In my case I use IPv6 over IPv4 and all works fine.
I'm using +- old RB750Gr3 on v6.49 with 1Gb EOIP tunnel over internet and recieve 800-870Mbps with IPv4 and ~890-900 with IPv6 over IPv4 (seems it works better?)

**Comment by alom4hdi** [comment]
+1 for ipv6 support 😍

**Comment by bbonev** [comment]
Check the latest release...

The ipv6 and keepalive and not fully test though

**Comment by marek22k** [comment]
Thanks for implementing IPv6 support!
